public_html/assets/css/nf-core.css

```diff
@@ -906,6 +920,29 @@ Based on https://codepen.io/wintr/pen/beBJBb */
   padding: 1rem 1rem 0.5rem;
 }
 
+.fa_icon_picker.popover {
+  width: 17rem;
+}
+.fa_icon_picker .popover-header input {
+  outline: none;
+  -webkit-box-shadow: none !important;
+  -moz-box-shadow: none !important;
+  box-shadow: none !important;
+  font-size: 0.8rem;
+}
+.fa_icon_picker .popover-header {
+  padding: 0;
+}
+.fa_icon_picker .popover-body {
+  height: 200px;
```

Comment:
found another one 🙂 (I can't test these changes locally and I am bad with guessing, so you have to fix that on your side)